xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm

```diff
@@ -179,26 +179,28 @@
   #end
   #template("hierarchy.vm")
   <div class="row">
-    #if ($displayContentMenu)
-      <div class="col-xs-12 col-md-5 pull-right">
-        #template("menus_content.vm")
-      </div>
-    #end
-    #if (($editor == 'wiki' || $editor == 'wysiwyg') && !$request.section)
-      <div id="editMeta" class="col-xs-12#if ($displayContentMenu) col-md-7#end">
-        #template('editmeta.vm')
-      </div>
-    #else
-      <div id="document-title" class="col-xs-12#if ($displayContentMenu) col-md-7#end"><h1>
-      #if($editor == 'rights')
-        $services.localization.render('core.editors.rights.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'object')
-        $services.localization.render('core.editors.object.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'class')
-        $services.localization.render('core.editors.class.title', [$escapetool.xml($doc.fullName), $doc.getURL()])
+    <dl>
+      #if ($displayContentMenu)
+        <div class="col-xs-12 col-md-5 pull-right">
+          #template("menus_content.vm")
```

Comment:
Is this template generating ``dt`` or ``dd`` elements? Otherwise the HTML is invalid. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl . You can use ``div`` inside a definition list, but the goal is to wrap ``dt`` and ``dd``.

Reply:
I don't understand why you mention ``editmeta.vm`` when the code above includes ``menus_content.vm`` which doesn't generate any ``dt`` or ``dd``. Do you see any example on the [Vertical Forms](https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/) documentation where the ``dl`` has **direct** child nodes other than ``dt`` and ``dd``?

Reply:
No there isn't direct child nodes to `dl` . Which I initially did. But it was creating another issue.
I'll see what I can do about this. But what about Content's `dd` (which is in another `div`) ?

Reply:
I can remove `dl` from here and add it around `#template('editmeta.vm')` but it will still be an invalid HTML. I can only use `dl` inside `editmeta.vm` to make a valid HTML. 
But again, content label gets unaligned.

Reply:
Anyways, I removed `dl` from here since it was not wrapping for a valid HTML in this file.

Reply:
You removed the ``dl`` but the code still appears as modified because you left the indentation. Make sure you have only relevant changes in the diff, otherwise it's hard to review.